docs/source/starter/installation.rst

```diff
@@ -67,16 +67,20 @@ If desired, the GeoIPS environment variables can be added to your
       # wget https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-x86_64.sh
       wget https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-Linux-x86_64.sh
 
-- Make the install script executable and run the installer,
-  following the prompts (particularly the bit about
-  conda init / restarting terminal!):
+- Make the installation script executable, and run the installer.
+  Accept the license terms and cofirm the location.
+- You would see a message from installer asking to close and
+  re-open the shell at the end of installation which is not required.
 
   .. code:: bash
 
       chmod u+x ./Mambaforge-Linux-x86_64.sh
-      # After installing, follow instructions regarding conda init / restarting your terminal !
+      # No terminal reconnect is required afterward.
       # Make sure you set env vars again if you restart your terminal !
       ./Mambaforge-Linux-x86_64.sh
+      # Run source ~/.bashrc to apply the Conda configurations.
+      source ~/.bashrc
```

Comment:
I hesitate to make this change. There are many things that can go wrong if someone has a badly constructed `.bashrc` and this is bash-specific. I think it is easier to just tell them to restart the terminal than it is to say "source the RC file for your current shell and if that seems to have screwed something up, try restarting your terminal session".

Reply:
The aim is to avoid closing the terminal, especially if the installation is on a remote machine. @jsolbrig, do you think using `exec bash` or `exec zsh` would work in our case? I plan to validate this approach whenever possible this week, but wanted to share the tentative approach here to avoid further exploration if a total different perspective emerges.

Reply:
Doesn't this assume that people put the conda init stage in the ~/.bashrc? I think this would only reset all the env variables